src/storage/DatedIrsVamm.sol

```diff
@@ -165,6 +175,30 @@ library DatedIrsVamm {
         self.mutableConfig.priceImpactBeta = _config.priceImpactBeta;
         self.mutableConfig.rateOracle = _config.rateOracle;
         self.mutableConfig.spread = _config.spread;
+
+        self.setMinAndMaxTicks(_config.minTick, _config.maxTick);
+    }
+
+    function setMinAndMaxTicks(
+        Data storage self,
+        int24 _minTick,
+        int24 _maxTick
+    ) internal {
+        if(
+            _minTick < TickMath.MIN_TICK_LIMIT || _maxTick > TickMath.MAX_TICK_LIMIT ||
+            self.vars.tick < _minTick || self.vars.tick > _maxTick
+        ) {
+            revert ExceededTickLimits(_minTick, _maxTick);
+        }
+
+        if(_minTick != -_maxTick) {
```

Comment:
minor point, `(_minTick + _maxTick != 0)` seems to be more readable.

Reply:
changed [60c86a9](https://github.com/Voltz-Protocol/v2-vamm/pull/33/commits/60c86a925e32609ebd7fbe4cd9802a749137d4fc)